src/components/Card.js

```diff
@@ -20,10 +32,16 @@ export default class Card {
     this._cardImage = this._template.querySelector("#card-URL");
 
     this._likeButton.addEventListener("click", () => {
+      if (this._isLiked) {
```

Comment:
Needs correction: You should change the likes button state and `isLiked` state only after the successful api response.

The code for rendering likes should look like this:

```js
// in Card class 

// call this method from index.js after successful api response in `then` block
setIsLiked(isLiked) { 
   // set instance variable
    this._isLiked = isLiked; 
    this._renderLikes(); 
} 

isLiked() {
  return this._isLiked;
}

_renderLikes() {
  if (this._isLiked) {
    // add active class to the like button
  } else {
    // remove active class from the like button
  }
}

createCard() { 
  ...
  // render initial likes, you saved them in the constructor
  this._renderLikes()
}

// index.js
if (card.isLiked()) {
  api.dislikeCard(...).then(response => card.setIsLiked(response._isLiked)).catch(() => /*handle error*/)
} else {
 // the same for adding like
}
```